global_helpers/panther_iocs.py

```diff
@@ -34,7 +34,663 @@
+    "147.182.131.229",
+    "146.56.131.161",
+    "145.220.24.19",
+    "143.198.45.117",
+    "143.198.32.72",
+    "143.110.221.204",
+    "142.93.36.237",
+    "142.93.34.250",
+    "140.246.171.141",
+    "139.59.8.39",
+    "137.184.111.180",
+    "137.184.106.119",
+    "137.184.104.73",
+    "137.184.102.82",
+    "137.184.99.8",
+    "137.184.98.176",
+    "137.184.96.216",
+    "137.184.28.58",
+    "135.148.43.32",
+    "134.122.34.28",
+    "134.56.204.191",
+    "133.18.201.195",
+    "122.161.53.44",
+    "122.161.50.23",
+    "121.5.219.20",
+    "121.4.56.143",
+    "116.24.67.213",
+    "109.237.96.124",
+    "109.70.100.36",
+    "109.70.100.34",
+    "109.70.100.28",
+    "109.70.100.27",
+    "107.189.29.107",
+    "107.189.14.182",
+    "107.189.14.98",
+    "107.189.14.76",
+    "107.189.12.135",
+    "107.189.11.153",
+    "107.189.10.137",
+    "107.189.8.65",
+    "107.189.1.178",
+    "107.189.1.160",
+    "104.244.79.6",
+    "104.244.78.213",
+    "104.244.76.173",
+    "104.244.76.170",
+    "104.244.76.13",
+    "104.244.75.74",
+    "104.244.74.211",
+    "104.244.74.57",
+    "104.244.73.43",
+    "104.244.72.129",
+    "104.244.72.115",
+    "104.244.72.7",
+    "103.214.5.13",
+    "103.103.0.142",
+    "103.103.0.141",
+    "103.90.239.209",
+    "101.35.154.34",
+    "94.230.208.147",
+    "94.142.241.194",
+    "91.245.81.65",
+    "91.219.237.21",
+    "91.203.5.146",
+    "89.249.63.3",
+    "89.163.252.230",
+    "89.163.154.91",
+    "88.80.20.86",
+    "82.221.131.71",
+    "81.17.18.62",
+    "81.17.18.61",
+    "81.17.18.60",
+    "68.183.198.247",
+    "68.183.44.143",
+    "68.79.17.59",
+    "66.220.242.222",
+    "64.113.32.29",
+    "62.171.142.3",
+    "62.102.148.68",
+    "62.76.41.46",
+    "61.19.25.207",
+    "60.31.180.149",
+    "45.153.160.133",
+    "23.154.177.2",
+    "23.129.64.145",
+    "23.120.182.121",
+    "5.199.143.202",
+    "185.220.101.191",
+    "185.220.101.186",
+    "185.220.101.185",
+    "185.220.101.168",
+    "185.220.101.163",
+    "185.130.44.108",
+    "185.129.61.4",
+    "185.129.61.1",
+    "185.107.47.215",
+    "185.107.47.171",
+    "185.100.87.202",
+    "185.100.87.41",
+    "185.83.214.69",
+    "185.56.80.65",
+    "185.38.175.132",
+    "181.214.39.2",
+    "178.176.203.190",
+    "178.176.202.121",
+    "178.62.79.49",
+    "178.20.55.16",
+    "178.17.171.150",
+    "178.17.171.102",
+    "176.10.104.240",
+    "176.10.99.200",
+    "175.6.210.66",
+    "172.106.17.218",
+    "138.68.167.19",
+    "62.102.148.69",
+    "46.166.139.111",
+    "23.129.64.141",
+    "109.70.100.26",
+    "185.220.102.8",
+    "185.220.100.254",
+    "185.220.100.252",
+    "72.223.168.73",
+    "51.255.106.85",
+    "46.182.21.248",
+    "23.129.64.148",
+    "23.129.64.146",
+    "23.129.64.131",
+     }
+
+LOG4J_EXPLOIT_IOCS = {
+     'jndi:ldap:/', 'jndi:rmi:/', 'jndi:ldaps:/', 'jndi:dns:/', 'jndi:nis:/', 'jndi:nds:/', 'jndi:corba:/', 'jndi:iiop:/'
```

Comment:
RE https://twitter.com/James_inthe_box/status/1470408273727668225?s=20, should we do `jndi:${l`

Reply:
Keep in mind, this won't catch https://twitter.com/Rezn0k/status/1469523006015750146?s=20

Reply:
> should we do jndi:${ 
Yeah thats a good call for sure, adding

> Tangentially, keep in mind, this won't catch all possible permutations
The amount of WAF / detection bypasses I'm seeing for this is insane

Reply:
Also adding `:${lower:`